apps/examples/testcase/le_tc/network/tc_net_ioctl.c

```diff
@@ -54,7 +54,7 @@ static void tc_net_ioctl_p(void)
 	}
 	long a = 0;
 
-	int ret = ioctlsocket(fd, FIONBIO, &a);
+	int ret = ioctl(fd, FIONBIO, (unsigned long)&a);
```

Comment:
How about declaring the type of the variable 'a' as unsigned long instead of just long in line 55?

Reply:
ok